docs/getting-started-guides/windows/index.md

```diff
@@ -1,139 +1,144 @@
+
+#### Overlay using OVN controller and OVS Switch Extension
 
-The following diagram illustrates the Windows Server networking setup for Kubernetes Setup:
-![Windows Setup](windows-setup.png)
+TODO
 
 ## Setting up Windows Server Containers on Kubernetes
-To run Windows Server Containers on Kubernetes, you'll need to set up both your host machines and the Kubernetes node components for Windows and setup Routes for Pod communication on different nodes.
+To run Windows Server Containers on Kubernetes, you'll need to set up both your host machines and the Kubernetes node components for Windows and depending on your network topology, setup Routes for Pod communication on different nodes.
 
 ### Host Setup
-**Windows Host Setup**
-
-1. Windows Server container host running Windows Server 2016 and Docker v1.12. Follow the setup instructions outlined by this blog post: https://msdn.microsoft.com/en-us/virtualization/windowscontainers/quick_start/quick_start_windows_server.
-2. DNS support for Windows recently got merged to docker master and is currently not supported in a stable docker release. To use DNS build docker from master or download the binary from [Docker master](https://master.dockerproject.org/).
-3. Pull the `apprenda/pause` image from `https://hub.docker.com/r/apprenda/pause`. 
-4. RRAS (Routing) Windows feature enabled.
-5. Install a VMSwitch of type `Internal`, by running `New-VMSwitch -Name KubeProxySwitch -SwitchType Internal` command in *PowerShell* window. This will create a new Network Interface with name `vEthernet (KubeProxySwitch)`. This interface will be used by kube-proxy to add Service IPs.
 
 **Linux Host Setup**
 
 1. Linux hosts should be setup according to their respective distro documentation and the requirements of the Kubernetes version you will be using. 
-2. CNI network plugin installed.
-
-### Component Setup
-
-Requirements
-
-* Git
-* Go 1.7.1+
-* make (if using Linux or MacOS)
-* Important notes and other dependencies are listed [here](https://git.k8s.io/community/contributors/devel/development.md#building-kubernetes-on-a-local-osshell-environment)
-
-**kubelet**
-
-To build the *kubelet*, run:
-
-1. `cd $GOPATH/src/k8s.io/kubernetes`
-2. Build *kubelet*
-   1. Linux/MacOS: `KUBE_BUILD_PLATFORMS=windows/amd64 make WHAT=cmd/kubelet`
-   2. Windows: `go build cmd/kubelet/kubelet.go`
-
-**kube-proxy**
-
-To build *kube-proxy*, run:
-
-1. `cd $GOPATH/src/k8s.io/kubernetes`
-2. Build *kube-proxy*
-   1. Linux/MacOS: `KUBE_BUILD_PLATFORMS=windows/amd64 make WHAT=cmd/kube-proxy`
-   2. Windows: `go build cmd/kube-proxy/proxy.go`
+2. Configure Linux Master node using steps [here](https://github.com/Microsoft/SDN/blob/master/Kubernetes/HOWTO-on-prem.md)
+3. [Optional] CNI network plugin installed.
```

Comment:
This is only partially true, right? By default (i.e. using `cni=kubenet`), you need at least the `host-local`, `loopback`, and `bridge` CNI plugins. And not every third-party CNI will work with Windows nodes. Maybe this is implied and known, but still thought I'd mention it.